CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+
+## [0.6.3-rc4]
+ - Changed so that Forge2DGame calls createBody from add instead of prepare
```

Comment:
I would personally change this sentence to be more of what the change really is:
```suggestion
 - Renamed the `prepare` method to `add` to be more inline with Flame Game naming.
```